_apps/topissues.md

```diff
@@ -0,0 +1,39 @@
+---
+# A human-friendly name of your listing
+title: Top Issues
+
+# A short description of what your app does
+description: Labels issues with the most "+1" emoji reactions
+
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: https://github.com/apps/top-issue
```

Comment:
This should be just `top-issue`, but it doesn't appear that you actually own the slug given https://github.com/apps/top-issue 404s

Reply:
`top-issues`.  updated.  

https://github.com/apps/top-issues-bot